command/agent/log_levels.go

```diff
@@ -10,7 +10,7 @@ import (
 // levels that we use.
 func LevelFilter() *logutils.LevelFilter {
 	return &logutils.LevelFilter{
-		Levels:   []logutils.LogLevel{"TRACE", "DEBUG", "INFO", "WARN", "ERR"},
+		Levels:   []logutils.LogLevel{"TRACE", "DEBUG", "INFO", "WARN", "ERROR"},
```

Comment:
this fixes an existing bug that doesn't allow specifying ERROR level logging as required by hclog (https://github.com/hashicorp/go-hclog/blob/master/logger.go#L59-L76)